src/frameworks/compose/driver/local.ts

```diff
@@ -18,12 +19,20 @@ export class LocalDriver implements Driver {
   }
 
   install(): void {
-    const [cmd, ...args] = this.spec.installCommand.split(" ");
+    let cmd = "";
+    let args: string[] = [];
+    if (this.spec.installCommand) {
```

Comment:
not sure why we run anything if `installCommand` is empty. Shouldn't this be:

```
if (this.spec.installCommand) {
  ...
  this.execCmd(cmd, args)
}
```

Reply:
we can avoid this